tests/environment.py

```diff
@@ -188,6 +188,19 @@ def _docker_wait_for_log(container: str, program: str, regex: str, start_line: i
         return (False, start_line, None)
 
 
+def _device_wait_for_log(device: None, log_path: str, regex: str,
+                         start_line: int, timeout: int) -> bool:
+    """Waits for log mathing regex expression to show up."""
+    device.sendline("tail -n10 -f {}".format(log_path))
```

Comment:
use long format in scripts. It increases readability and requires less efforts from a newcomer
```suggestion
    device.sendline("tail --lines=10 --follow {}".format(log_path))
```

Reply:
Unfortunately, [long options are non-POSIX](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/tail.html#tag_20_125_04) so we can't be sure that the `tail` implementation on target supports them. I believe in the busybox implementation, the long option support is optional.

Reply:
Sadly, but `tail` of `busybox` used in RAX40 images doesn't support `--lines` option. But, i was checking pre-RC1 image of prplWRT on RAX40.

Reply:
Resolving, as there is no way for us to use long options, as they are not supported on RAX40.